court_scraper/platforms/wiscourts/site.py

```diff
@@ -0,0 +1,16 @@
+from court_scraper.case_info import CaseInfo
+from court_scraper.base.selenium_site import SeleniumSite
+
+from .pages.case_detail import CaseDetails
+from .pages.search import SearchPage
+from court_scraper.base._selenium_site_search_mixin import SiteSearchMixIn
+
+class WiscourtSite(SeleniumSite, SiteSearchMixIn):
```

Comment:
Again, let's make the class name match the module name. If you want the class to have a more friendly "importable name", you could do an `from .site import Site as WiscourtSite` in `wiscourts/__init__.py`